scripts/config/msu/text.nut

```diff
@@ -13,4 +13,23 @@
 	{
 		return this.color(::Const.UI.Color.NegativeValue, _string);
 	}
+
+	function colorizeValue( _value, _invert = false, _addSign = true )
+	{
+		if (_value < 0) return this[_invert ? "colorGreen" : "colorRed"](_value);
+
+		if (_value > 0) return (_addSign ? "+" : "") + this[_invert ? "colorRed" : "colorGreen"](_value);
+
+		if (_value == 0) return this[_invert ? "colorRed" : "colorGreen"](_value);
```

Comment:
I would just make it optionally not color at all in this case (another parameter lol)

Reply:
I don't see the point of it. Could you elaborate on why it is useful?

Reply:
If you're showing a range of stuff you may want 0 to be neutral (and therefore black)

Reply:
In that case you wouldn't use this function on the first part of the range. Because there is no way to turn the colorization on/off based on the value.